lib/runtime/wazero/test_helpers.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 ChainSafe Systems (ON)
```

Comment:
can we rename this file to `helpers_test.go` then they will be included only in the test build

Reply:
I can't because the functions used in this file are littered throughout the codebase.  We'd have to move these functions into a separate runtime helpers package, but I still can't underscore it with test, since it's a separate package.